### PR TITLE
add destination_account field to PathOption(path_find method)

### LINKED ISSUE #2159: Missing destination_amount field to path_find Response Format
If `send_max` is set in the `path_find` method, the 
The `alternatives` field (array) of the response will contain a `destination_amount` field.

related: https://github.com/XRPLF/xrpl-dev-portal/issues/1626

[WebSocket Tool](https://xrpl.org/websocket-api-tool.html?server=wss%3A%2F%2Fs1.ripple.com%2F&req=%7B%22id%22%3A8%2C%22command%22%3A%22path_find%22%2C%22subcommand%22%3A%22create%22%2C%22source_account%22%3A%22rQQQrUdN1cLdNmxH4dHfKgmX5P4kf3ZrM%22%2C%22destination_account%22%3A%22r9C47nTV3aQazuUwtA2FCyoxwdzs5U65pR%22%2C%22destination_amount%22%3A%7B%22currency%22%3A%22CSC%22%2C%22issuer%22%3A%22rCSCManTZ8ME9EoLrSHHYKW8PPwWMgkwr%22%2C%22value%22%3A%22-1%22%7D%2C%22send_max%22%3A%22100000%22%7D)
```json
{
  "id": 8,
  "command": "path_find",
  "subcommand": "create",
  "source_account": "rQQQrUdN1cLdNmxH4dHfKgmX5P4kf3ZrM",
  "destination_account": "rQQQrUdN1cLdNmxH4dHfKgmX5P4kf3ZrM",
  "destination_amount": {
      "currency": "CSC",
      "issuer": "rCSCManTZ8ME9EoLrSHHYKW8PPwWMgkwr",
      "value": "-1"
  },
  "send_max": "100000"
}
```

```json
{
  "forwarded": true,
  "id": 8,
  "result": {
    "alternatives": [
      {
        "destination_amount": {
          "currency": "CSC",
          "issuer": "rCSCManTZ8ME9EoLrSHHYKW8PPwWMgkwr",
          "value": "225.806180567527"
        },
        "paths_computed": [
          [
            {
              "currency": "CSC",
              "issuer": "rCSCManTZ8ME9EoLrSHHYKW8PPwWMgkwr",
              "type": 48
            }
          ]
        ],
        "source_amount": "100000"
      }
    ],
    "destination_account": "rQQQrUdN1cLdNmxH4dHfKgmX5P4kf3ZrM",
    "destination_amount": {
      "currency": "CSC",
      "issuer": "rCSCManTZ8ME9EoLrSHHYKW8PPwWMgkwr",
      "value": "-1"
    },
    "full_reply": false,
    "id": 8,
    "source_account": "rQQQrUdN1cLdNmxH4dHfKgmX5P4kf3ZrM"
  },
  "status": "success",
  "type": "response",
  "warnings": [
    {
      "id": 1004,
      "message": "This is a reporting server.  The default behavior of a reporting server is to only return validated data. If you are looking for not yet validated data, include \"ledger_index : current\" in your request, which will cause this server to forward the request to a p2p node. If the forward is successful the response will include \"forwarded\" : \"true\""
    }
  ]
}
```

### PR DESCRIPTION
**Pull request by develoQ**
## High Level Overview of Change

fix #2159 

### Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### CODE REVIEW

File: packages/xrpl/src/models/methods/pathFind.ts [hunk 1]
```diff
@@ -63,6 +63,11 @@ interface PathOption {
    * Destination to receive the desired amount.
    */
   source_amount: Amount
+  /**
+   * Destination Amount that the destination would receive along this path for the.
+   * If the `send_max` field is set, this field will be set.
+   */
+  destination_account?: Amount
```
Comment (mvadari) [inline packages/xrpl/src/models/methods/pathFind.ts:70]:
```suggestion
  destination_amount?: Amount
```

Comment (develoQ) [reply packages/xrpl/src/models/methods/pathFind.ts]: It was a small mistake. Thanks.

**Review by pdp2121 — APPROVED** [review]

File: packages/xrpl/src/models/methods/pathFind.ts [hunk 2]
```diff
@@ -63,6 +63,11 @@ interface PathOption {
    * Destination to receive the desired amount.
    */
   source_amount: Amount
+  /**
+   * Destination Amount that the destination would receive along this path for the.
```
Comment (JST5000) [inline packages/xrpl/src/models/methods/pathFind.ts:67]:
```suggestion
   * Destination Amount that the destination would receive along this path.
```

**Review by JST5000 — APPROVED** [review]
LGTM

**Pull request merged**